client/post-editor/media-modal/edit/edit-toolbar.jsx

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
The whole component should be better namespaced at the folder level. Something like `/media-modal/edit-image`